session/pingpong/accountant_caller.go

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2019 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package pingpong
+
+import (
+	"github.com/mysteriumnetwork/node/requests"
+	"github.com/mysteriumnetwork/payments/crypto"
+	"github.com/pkg/errors"
+)
+
+// AccountantCaller represents the http caller for accountant
+type AccountantCaller struct {
+	transport         requests.HTTPTransport
+	accountantBaseURI string
+}
+
+// NewAccountantCaller returns a new instance of accountant caller
+func NewAccountantCaller(transport requests.HTTPTransport, accountantBaseURI string) *AccountantCaller {
+	return &AccountantCaller{
+		transport:         transport,
+		accountantBaseURI: accountantBaseURI,
+	}
+}
+
+// RequestPromise requests a promise from accountant
+func (ac *AccountantCaller) RequestPromise(em crypto.ExchangeMessage) (crypto.Promise, error) {
+	req, err := requests.NewPostRequest(ac.accountantBaseURI, "/request_promise", em)
+	if err != nil {
+		return crypto.Promise{}, errors.Wrap(err, "could not form request_promise request")
+	}
+	var resp crypto.Promise
```

Comment:
Can it be changed to return pointer `*crypto.Promise`? I saw there is crypto.CreatePromise which returns pointer so maybe we should keep it the same.